xml/tuning-os-tuning.xml

```diff
@@ -283,10 +263,7 @@ salt '*' state.apply my_kerntune
 salt '*' cmd.run 'ip link set bond0 mtu 9000'
 </screen>
           <para>
-            To set this persistently, use YaST to set the MTU for the bonded interface
-            or adjust the <filename>/etc/sysconfig/network/ifcfg-*</filename> file's MTU settings
-            and restart the network. Ensure that jumbo frames are enabled on all
-            switches, routers, and network cards that will transport these packets.
+            To set this persistently, utilize YaST to set the MTU for the bonded interface.
```

Comment:
```suggestion
            To set this persistently, utilize &yast; to set the MTU for the bonded interface.
```

Reply:
Additional changes in older text:

>  We recommend to test at least random 4&nbsp;kB, random 64&nbsp;kB, and sequential

 We recommend **testing** at least random 4&nbsp;kB, random 64&nbsp;kB, and sequential

>  any caches that may apply. We recommend to use the <literal>ramp-time</literal>

 any caches that may apply. We recommend **using** the <literal>ramp-time</literal>

>  and needs to only be done in conjunction with the

 and needs to be done **only** in conjunction with the